manim/animation/transform.py

```diff
@@ -144,6 +144,68 @@ def interpolate_submobject(
 
 
 class ReplacementTransform(Transform):
+    """Replace the mobject with target_mobject (like Transform does), and shows the target_mobject (whereas Transform shows the mobject)
```

Comment:
The distinction between `ReplacementTransform` and `Transform` would be better suited as a note.
```suggestion
    """Replaces and morphs a mobject into the target mobject.
```